### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by chieger** [Rudko/Carousel#1]
@Rudko I have a feeling that somehow there is being a new alert action created and attached each time the the button is pressed 🤔

**Comment by Rudko** [comment]
@chieger  You're right! I think now I understand, where is the problem, I'll fix it!
